### DISCUSSION TITLE
New ground vehicle requests? Coming soon?

### DISCUSSION THREAD

**Discussion by KLABDB3K** [PygmalionOfCyprus/cmo-db-requests#1219] [Q&A] [answered]
Since Tiny beta there is clearly a direction of travel for Ground vehicles as seperate entities, which is brilliant. 
Can we expect a ground vehicle request template?
There are clearly works in progress for many of the ground vehicles currently in the Db493 so is there a timescale or plan for these to be resolved? Seems no point in making amendment requests when its clearly a work in progress?
Thanks. K

**Comment by PygmalionOfCyprus** [comment, marked as answer]
It's funny you should ask: @Jkryos and I were just discussing this at our last weekly meeting.

As you mentioned, the Ground Units annex is "clearly a work in progress." We came to the same conclusion you did: that is, there's no point taking requests at this stage. We know what's missing!

Our top priority right now is to finish filling out contemporary units for the "big players" (Russia, China, USA, etc.) -- not exhaustively, but deep enough that 90% of the platforms anyone might want to use are in there. This is where the greatest demand is pro-side, and I suspect it will sate a large number of commercial users as well.

After that, we're going to handle the "punch above their weight" countries: UK, France, Germany, etc. whose equipment is either unique (e.g. the Leopard II, which can't be modeled with a generic T-72 or something), heavily represented internationally (Austria, Sweden, etc.), relevant to modern flashpoints (hello, Japan), or all of the above. That should give us a good springboard from which to begin filling out smaller countries without strong domestic arms industries: having a strong foundation of "popular" units will give us plenty of options to use as baseline entries for new platforms and therefore exponentially increase the speed at which we can add new units.

Finally, we'll do a "historical" pass. There's no way we'll be able to add every historic tank, APC, IFV, etc. but we can at least make sure the "big guys" are in there, if they haven't already been added during the above phases.

Only once we've achieved all that do I think it would be productive for us to take specific Ground Units requests.

**Reply by KLABDB3K** [reply]
Good plan. Thanks for all your efforts.

**Reply by claudejdev** [reply]
Hey there @PygmalionOfCyprus  and @Jkryos,

just want to mention that I have references about vehicles' amphibious speed and such (Jane's).
I'm ready to post all that in a table, maybe so you guys know at a glance what are the "known unknowns".



Questions :

> Our top priority right now is to finish filling out contemporary units for the "big players" (Russia, China, USA, etc.) -- not exhaustively, but deep enough that 90% of the platforms anyone might want to use are in there. This is where the greatest demand is pro-side, and I suspect it will sate a large number of commercial users as well.
> 
> After that, we're going to handle the "punch above their weight" countries: UK, France, Germany, etc. whose equipment is either unique (e.g. the Leopard II, which can't be modeled with a generic T-72 or something), heavily represented internationally (Austria, Sweden, etc.),

- Q1 : Are you talking specifically about ground units here, or is the SOP for all platforms?

- Q2 : what about the soon-to-be pro-side (bonjour, hopefully)? And even current "punch above their weight" pro, such as Germany?

- Q3 : does the OSINT data from the pro-side DB percolates to the commercial DB, or is it the other way around only?
